### PR TITLE
Adapt remote (fsspec) file reading for newer pytroll-collector changes

### PR DESCRIPTION
**Pull request by mraspaud**
Recent changes in the pytroll-collector package regarding formatting of message for remote filesystems require trollflow2 to adapt. This PR implements the adaptation.

 - [x] Tests added

### CODE REVIEW

File: trollflow2/launcher.py [hunk 1]
```diff
@@ -266,7 +266,7 @@ def _extract_filenames(msg):
 
     If the message contains a `filesystem` item, use fsspec to decode it.
     """
-    filenames = [urlparse(uri).path for uri in gen_dict_extract(msg.data, 'uri')]
+    filenames = [uri.split("::")[0] for uri in gen_dict_extract(msg.data, 'uri')]
```
Comment (pnuu) [inline trollflow2/launcher.py:269]: Wouldn't this return just `'simplecache'` if the URI is something like `simplecache::s3://noaa-goes17/ABI-L1b-RadF/2021/195/20/*_G17_s20211952030*`?

**Review by pnuu — COMMENTED** [review]
Just one question.

File: trollflow2/launcher.py (hunk 1 above)
Comment (adybbroe) [reply trollflow2/launcher.py:269]: Seems reasonable @pnuu , perhaps we should add a test for that then?

Comment (mraspaud) [reply trollflow2/launcher.py:269]: Looking at it now

**Review by pnuu — APPROVED** [review]
LGTM

**Pull request merged**